.github/workflows/trigger_rebase.yml

```diff
@@ -0,0 +1,42 @@
+name: Trigger Rebase
+
+on:
+  push:
```

Comment:
Does the `push` event trigger when a branch head is moved by merging a PR via the GitHub site? The `develop` branch does not accept external pushes so I wasn't sure what this event covers.

Reply:
Yep, the CI gets run on develop when we merge PRs. This reponse in the Github Community confirms the same too https://github.community/t/does-a-pull-request-merge-always-trigger-the-push-event/17032/2?u=pezholio